book/src/external-provisioner.md

```diff
@@ -10,15 +10,16 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](project-policies.md#minimum-version) | [Max K8s Version](project-policies.md#maximum-version) | [Recommended K8s Version](project-policies.md#recommended-version) |
 --|--|--|--|--|--|--|--
-[external-provisioner v3.5.0](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v3.4.1) | [release-3.4](https://github.com/kubernetes-csi/external-provisioner/tree/release-3.5) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | registry.k8s.io/sig-storage/csi-provisioner:v3.5.0 | v1.20 | - | v1.26
+[external-provisioner v3.6.0](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v3.6.0) | [release-3.4](https://github.com/kubernetes-csi/external-provisioner/tree/release-3.6) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | registry.k8s.io/sig-storage/csi-provisioner:v3.6.0 | v1.20 | - | v1.28
+[external-provisioner v3.5.0](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v3.5.0) | [release-3.4](https://github.com/kubernetes-csi/external-provisioner/tree/release-3.5) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | registry.k8s.io/sig-storage/csi-provisioner:v3.5.0 | v1.20 | - | v1.26
 [external-provisioner v3.4.1](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v3.4.1) | [release-3.4](https://github.com/kubernetes-csi/external-provisioner/tree/release-3.4) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | registry.k8s.io/sig-storage/csi-provisioner:v3.4.1 | v1.20 | - | v1.26
 [external-provisioner v3.3.1](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v3.3.1) | [release-3.3](https://github.com/kubernetes-csi/external-provisioner/tree/release-3.3) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | registry.k8s.io/sig-storage/csi-provisioner:v3.3.1 | v1.20 | - | v1.25
-[external-provisioner v3.2.2](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v3.2.2) | [release-3.2](https://github.com/kubernetes-csi/external-provisioner/tree/release-3.2) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | registry.k8s.io/sig-storage/csi-provisioner:v3.2.2 | v1.20 | - | v1.22
```

Comment:
v3.3.0 was released on 9/26/2022. So 3.3.x has been out for a year. 3.3.1 can be moved to unsupported table.

Reply:
my bad